demo/remote/terraform/env/aws/terraform.tfvars

```diff
@@ -0,0 +1,6 @@
+region             = "us-east-1"
```

Comment:
in the past, i've seen folks make this a `terraforms.tfvars.sample` file and exclude tfvars using gitignore. i don't know if this is standard practice (to avoid accidentally committing tfvars) or not. just noting that.

Reply:
that's a neat idea; i'll update!